src/Sylius/Bundle/AdminBundle/Resources/views/Customer/Field/createdAt.html.twig

```diff
@@ -0,0 +1,3 @@
+{% if data.user is not null %}
+{{ data.user.createdAt|date("d-m-Y H:i") }}
```

Comment:
I recently merged @Zales0123's PR, which introduced ``datetime`` grid column type, let's use it here instead of twig.